en/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -3,47 +3,193 @@ id: applications-databases-elasticsearch
+| active_shards_percent    | Percentage of active shards              |    %    |
+| tasks_pending            | Number of pending tasks                  |         |
+| docs_count               | Number of documents                      |         |
+| size_in_bytes            | Size of all shards assigned              |    B    |
+
+<!--Indice-statistics-->
+
+| Metric name              | Description                              | Unit    |
+| :----------------------- | :--------------------------------------- | :------ |
+| display                  | Name of the Elasticsearch Indice         |         |
+| status                   | Status of the Elasticsearch Indice       |         |
+| shards_active            | Number of active shards                  |         |
+| shards_unassigned        | Number of unassigned shards              |         |
+| docs_count               | Number of documents                      |         |
+| size_in_bytes_primaries  | Size of all primary shards               |    B    |
+| size_in_bytes_total      | Total size of all shards assigned        |    B    |
+
+<!--License-->
+
+| Metric name        | Description                              | Unit    |
+| :----------------- | :--------------------------------------- | :------ |
+| type               | Licence type                             |         |
+| status             | Licence status                           |         |
+| issued_to          | Licence owner                            |         |
+| issue_date         | Licence issue date                       |         |
+
+<!--Node-statistics-->
+
+| Metric name       | Description                                           | Unit    |
+| :---------------- | :---------------------------------------------------- | :-----  |
+| display           | Name of the Elasticsearch node                        |         |
+| indices_count     | Number of indices in the node                         |         |
+| heap_used_percent | Percentage of memory currently in use by the heap     |    %    |
+| heap_used_in_bytes| Amount of memory available for use by the heap        |    B    |
+| heap_max_in_bytes | Maximum amount of memory available for use by the heap|    B    |
+| available_in_bytes| Total number of bytes available                       |    B    |
+| total_in_bytes    | Total size of all file stores                         |    B    |
+| docs_count        | Number of documents inside the indice                 |         |
+| size_in_bytes     | Total size of all shards assigned to the node         |    B    |
+
+<!--Rules-->
+
+| Rule name                              | Description                                   |
+| :------------------------------------- | :-------------------------------------------- |
+| App-DB-Elasticsearch-Indice-Statistics | Discover the indices on your Elasticsearch DB |
+| App-DB-Elasticsearch-Node-Statistics   | Discover the nodes on your Elasticsearch DB   |
```

Comment:
Put this section at the beginning like this :
```suggestion
### Monitored objects
[...]

### Discovery rules

<!--DOCUSAURUS_CODE_TABS-->

<!--Services-->

| Rule name                              | Description                                   |
| :------------------------------------- | :-------------------------------------------- |
| App-DB-Elasticsearch-Indice-Statistics | Discover the indices on your Elasticsearch DB |
| App-DB-Elasticsearch-Node-Statistics   | Discover the nodes on your Elasticsearch DB   |
```
<!--END_DOCUSAURUS_CODE_TABS-->